<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueSetW</class>
 <widget class="QDialog" name="ValueSetW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ValueSetW
{
	border-image: url(:/new/prefix1/image/locktimeimag/back_board.png);
}</string>
  </property>
  <widget class="QGroupBox" name="titleGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>980</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleGroupBox
{
	border-image: url(:/new/prefix1/image/locktimeimag/back_board1.png);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="returnParentButton">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>27</y>
      <width>16</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/locktimeimag/return.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLable">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>11</y>
      <width>191</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(76, 76, 76);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>锁屏时间</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>237</y>
     <width>380</width>
     <height>330</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox
{
	border-image: url(:/new/prefix1/image/locktimeimag/board.png);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/locktimeimag/pbut_add.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/locktimeimag/pbut_add.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/locktimeimag/pbut_add1.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lessButton">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/locktimeimag/pbut_less.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/locktimeimag/pbut_less.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/locktimeimag/pbut_less1.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/locktimeimag/ok.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/locktimeimag/ok.png)}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 255);</string>
    </property>
    <property name="text">
     <string>单位：/分钟</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2
{
	border-image: url(:/new/prefix1/image/locktimeimag/data_board.png);
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
